Sources/MessagingInApp/Gist/Gist.swift

```diff
@@ -121,10 +121,6 @@ public class Gist: GistInstance, GistDelegate {
         delegate?.action(message: message, currentRoute: currentRoute, action: action, name: name)
     }
 
-    public func embedMessage(message: Message, elementId: String) {
```

Comment:
`embedMessage` function call [removed in this commit](Sources/MessagingInApp/Gist/Managers/MessageQueueManager.swift). Because it's no longer needed, I removed it here. 

This is a public function of Gist, but it's not part of our advertised public API. So removing it should not be considered a breaking change to our SDK.

Reply:
I think we still need to provide customers with a way to add messages locally, we might need to check with in-app squad about this. We could probably give a method on top of `InAppMessageView`.  Doesn't need to happen in this PR but maybe a ticker or todo so we don't forget.

Reply:
Since this feature isn't on the project roadmap, I'll hold off on creating a ticket or todo. If you think it's important, please pitch it to the team, perhaps in the project Slack channel.